plinth/__main__.py

```diff
@@ -285,6 +292,17 @@ def list_dependencies(module_list):
     sys.exit(error_code)
 
 
+def list_modules() :
+    """List all enabled modules"""
+    for module_name, module in module_loader.loaded_modules.items():
+        print('{module_name}'.format(module_name=module_name))
+
+def list_modules_essential() :
+    for module_name, module in module_loader.loaded_modules.items(essential=True):
```

Comment:
If the module is essential, then module.is_essential should be True.

However, not all modules define is_essential. So we can use getattr(module, 'is_essential', False), which will return False if it's not defined.